### DISCUSSION TITLE
What is the unit for "rain"?

### DISCUSSION THREAD

**Discussion by MichalBryxi** [winds-mobi/winds-mobi-providers#95] [Writing your own provider] [answered]
When looking at the weather stations they have "Rain: xxx `l/m²`", but that unit seems to be missing a dimension of time. Is that per hour / day / month?

I'm mainly asking because I want to bring in the [rain data for pmcjoder](https://www.pmcjoder.ch/webcam/neuhaus/wetterstation/details.htm) weather stations: 

![Screenshot 2024-06-30 at 19 15 34](https://github.com/winds-mobi/winds-mobi-providers/assets/847473/f2dfcd6f-5670-4ec4-89d1-906be655a83a)

**Comment by ysavary** [comment, marked as answer]
Yes, by default the rain unit is `liter/m2`: https://github.com/winds-mobi/winds-mobi-providers/blob/main/winds_mobi_provider/provider.py#L139. I assume it is liter per square meter during the last measurement period (like the wind maximum of the latest period).

You can use another unit by passing a [Pint](https://github.com/hgrecco/pint) value to [create_measure()](https://github.com/winds-mobi/winds-mobi-providers/blob/main/winds_mobi_provider/provider.py#L379).
